rotkehlchen/tests/api/test_data_import.py

```diff
@@ -215,6 +219,12 @@ def test_data_import_nexo(
     dir_path = Path(__file__).resolve().parent.parent
     filepath = dir_path / 'data' / 'nexo.csv'
 
+    # add location mapping for luna.
```

Comment:
what's this?

Reply:
so from the entry, unknown asset is raised due to a missing mapping -- that's what i added it here.

there's a pr to update location mappings too.

Reply:
The entry? What do you mean? What entry? Please can you discuss things in discord and communicate faster?

If you mean the test data you are using, given by the user? Isn't the solution to add the mapping to the global DB and not manually in a single test?